examples/run_generation.py

```diff
@@ -93,15 +95,16 @@ def prepare_ctrl_input(args, _, tokenizer, prompt_text):
     return prompt_text
 
 
-def prepare_xlm_input(args, model, tokenizer, prompt_text):
+def prepare_xlm_input(model, tokenizer, prompt_text, **kwargs):
     # kwargs = {"language": None, "mask_token_id": None}
 
     # Set the language
+    xlm_language = kwargs.get("xlm_language")
```

Comment:
Any reason to explicitly `get`? Why not simply do `if kwargs['xlm_language'] in available_languages`? It would ave you an extra line. IIRC the idea of using args + namespaces was to get the full possible benefit of quick attribute access and autocompletion in some editors. Not sure about the latter. Reverting back to default kwargs might not be desirable.